dolphinscheduler-python/pydolphinscheduler/docs/source/tutorial.rst

```diff
@@ -38,7 +38,7 @@ There are two types of tutorials: traditional and task decorator.
   supported. But it is helpful if your workflow is all built with Python or if you already have some Python
   workflow code and want to migrate them to pydolphinscheduler.
 - **YAML File**: We can use pydolphinscheduler CLI to create process using YAML file: :code:`pydolphinscheduler yaml -f tutorial.yaml`. 
-  We can find more YAML file examples in `examples/yaml_define <https://github.com/apache/dolphinscheduler/tree/py-yaml/dolphinscheduler-python/pydolphinscheduler/examples/yaml_define>`_
+  We can find more YAML file examples in `examples/yaml_define <https://github.com/apache/dolphinscheduler/tree/dev/dolphinscheduler-python/pydolphinscheduler/examples/yaml_define>`_
```

Comment:
we should not use whole link in `rst` file, we should better use `:ref:` syntax for hyper link. Such as https://github.com/apache/dolphinscheduler/blob/3025f67d370e02ebae075a4ef68e1d7f0461b2d6/dolphinscheduler-python/pydolphinscheduler/docs/source/cli.rst#L26 is show 

```rst
:ref:`install PyDolphinScheduler <start:installing pydolphinscheduler>`
```